src/app/containers/ATIAnalytics/index.jsx

```diff
@@ -31,6 +32,7 @@ const ATIAnalytics = ({ data }) => {
 };
 
 ATIAnalytics.propTypes = {
-  data: oneOfType([articleDataPropTypes]).isRequired,
+  data: oneOfType([articleDataPropTypes, shape(frontPageDataPropTypes)])
```

Comment:
this looks kinda weird, should one of these be changed to include/remove the shape()?

Reply:
I only did this because `frontPageDataPropTypes` is also used in FrontPageMain container, which means also updating the value there. 

I didn't really have a preference here, but I'm happy to update it